backend/src/SchemaFetcher.ts

```diff
@@ -0,0 +1,27 @@
+import { SanityApiResponse, SanityElement } from 'types/sanity';
+import { makeApiRequest } from './axiosClient';
+
+export async function getSchemaInformation(): Promise<string> {
+  let tagedString: string = '';
+  try {
+    const response = await makeApiRequest<SanityApiResponse>('', {
+      query: '*[_type =="introduction"]',
+    });
+
+    if (response.status === 200 && response.data.result) {
```

Comment:
Values of the array are incorrect - the map returns too many separate `strong` tags